DEVELOPMENT.md

```diff
@@ -57,8 +57,8 @@ Before running the Example app you will want to add configuration for a valid To
 private let e3dbToken = "259918a1804c89f4acd10b134810442284e4b2d93776c5a7a1c8934e14113a81"
 ```
 
-## Building (Release 4.2.2)
-Due to difficulties using Carthage as a package manager for the project, for release 4.2.2 the SDK has transitioned to using [Cocoapods](https://cocoapods.org/) for both the `E3db` SDK and `E3db-example` test application projects in this repository. The current build instructions for version 4.2.2 and beyond are detailed here.
+## Building (Release 4.2.3)
+Due to difficulties using Carthage as a package manager for the project, for release 4.2.3 the SDK has transitioned to using [Cocoapods](https://cocoapods.org/) for both the `E3db` SDK and `E3db-example` test application projects in this repository. The current build instructions for version 4.2.3 and beyond are detailed here.
```

Comment:
I'd probably rephrase all this to just say "As of 4.2.2, the Carthage package manager is no longer supported. The SDK should be install with Cocoapods(link)."